src/components/dialog/dialog.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/_variables.scss';
+
+.modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5); //Backdrop
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 999;
+}
+
+.modal-content {
+  background-color: $background-color;
+  border-radius: 5px;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.modal-title {
+  color: $icon-color;
+  font-size: 24px;
+  margin-bottom: 12px;
+}
+
+
+.modal-body {
+  padding: 0px 24px 24px 24px;
+}
+
+.close {
+  cursor: pointer;
+  font-size: 3.2rem;
+  color: $primary-color;
+  font-weight: 100;
+  margin-right: 8px;
+  padding: 0px;
+  line-height: 3rem;
+}
```

Comment:
Nest Parent -> Child

Reply:
Nested style according to html structure.